package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gpii-no-os",
+  "description": "Stub definitions suitable for booting up the GPII Morphic UI implemented in gpii-app without platform-specific support",
+  "version": "0.3.0",
+  "author": "Raising the Floor -- US Inc.",
+  "bugs": {
+    "url": "http://issues.gpii.net/browse/GPII"
+  },
+  "homepage": "http://gpii.net/",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Please make this a no-op which returns true so that we stand a chance of enlisting this in CI : P

Reply:
Sorry, obviously I mean "0" rather than true : P

Reply:
Understood.  npm test will now execute "exit 0" for the stubs.